ee/src/main/java/org/jboss/as/ee/subsystem/EeSubsystemRootResource.java

```diff
@@ -46,7 +46,7 @@ public class EeSubsystemRootResource extends SimpleResourceDefinition {
 
     public static final String WILDFLY_NAMING = "org.wildfly.naming";
     public static final String JBOSS_INVOCATION = "org.jboss.invocation";
-    public static final String JSON_API = "javax.json.api";
+    public static final String JSON_API = "jakarta.json.api";
     public static final String GLASSFISH_EL = "org.glassfish.javax.el";
```

Comment:
`org.glassfish.javax.el` is an alias of `org.glassfish.jakarta.el` which is indeed an alias for `org.glassfish.expressly`, but we are taking into account a limited set of aliases to avoid incompatibilities via exclusions, so I intentionally left it as it is.